src/createRollupConfig.ts

```diff
@@ -195,18 +196,22 @@ export async function createRollupConfig(
       //   printInfo: false,
       // }),
       shouldMinify &&
-        terser({
-          sourcemap: true,
-          output: { comments: false },
-          compress: {
-            keep_infinity: true,
-            pure_getters: true,
-            passes: 10,
-          },
-          ecma: 5,
-          toplevel: opts.format === 'cjs',
-          warnings: true,
-        }),
+        (opts.closureCompiler
+          ? closureCompiler({
+              compilation_level: 'SIMPLE_OPTIMIZATIONS',
+            })
```

Comment:
this is the default, right? so can just be:
```suggestion
          ? closureCompiler()
```

Reply:
yea, fixed